src/containers/SortingView.tsx

```diff
@@ -130,16 +130,16 @@ const SortingView: React.FunctionComponent<Props> = (props) => {
 
   const handleUnitClicked = useCallback((unitId, event) => {
     if (event.ctrlKey) {
-      if (selection.selectedUnitIds.includes(unitId)) {
-        selectionDispatch({type: 'SetSelectedUnitIds', selectedUnitIds: selection.selectedUnitIds.filter(uid => (uid !== unitId))})
+      if ((selection.selectedUnitIds || []).includes(unitId)) {
```

Comment:
Since the fields on selection are all optional, we need to do a lot of '|| []' throughout. This could be avoided by making a `const selectedUnitIds = selection.selectedUnitIds || []` at the top of these functions. Will do that sometime in the future.

Reply:
i've been avoiding the `?.` because I got compiler errors when trying to compile the jupyter extension. However, I think I've resolved that issue now, and so my avoidance of `?.` is purely a habit right now. Will revisit this.